### DISCUSSION TITLE
How to Checkif the number exist in whatsapp??

### DISCUSSION THREAD

**Discussion by Praveensenpai** [3mora2/WPP_Whatsapp#28] [Q&A] [answered]
I tried sending message to 91234 it does send but I don't think it's valid number

I also tried sending 81818181811818181 it's sending but i don't think it's valid number

so any way to valid it first to checkif the n;umber exist in whatsapp or not  before sending it???

**Comment by 3mora2** [comment, marked as answer]
See [SendMessageOptions](https://wppconnect.io/wa-js/interfaces/chat.SendMessageOptions.html#createChat)

See [all WhatsApp method](https://wppconnect.io/wppconnect/classes/Whatsapp.html)
